### PR TITLE
Increase max number of open files

### PR DESCRIPTION
**Pull request by andy-mishechkin**
According to https://github.com/actions/virtual-environments/issues/268, we have modified Ubuntu 16/18 images for adding the [DefaultLimitNOFILE=50000] setting to /etc/systemd/system.conf.
This setting will be used in non-interactive sessions for increasing the max open files limit

### CODE REVIEW

File: images/linux/ubuntu1604.json [hunk 1]
```diff
@@ -79,7 +79,8 @@
                 "systemctl disable apt-daily-upgrade.service",
                 "echo '* soft nofile 50000 \n* hard nofile 50000' >> /etc/security/limits.conf",
                 "echo 'session required pam_limits.so' >> /etc/pam.d/common-session",
-                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive"
+                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive",
+                "echo 'DefaultLimitNOFILE=50000' >> /etc/systemd/system.conf"
```
Comment (sergey-akhalkov) [inline images/linux/ubuntu1604.json]: Just wondering why we should not use Max possible number, what the motivation to set 50000?

Comment (andy-mishechkin) [reply images/linux/ubuntu1604.json]: Previous max file limit just has been repeated in this setting

Comment (sergey-akhalkov) [reply images/linux/ubuntu1604.json]: Let's change it to the max possible value if you see not issues with it.

Comment (andy-mishechkin) [reply images/linux/ubuntu1604.json]: Max files limit has been changed to 65536

**Pull request merged**